app/components/bread-crumb.js

```diff
@@ -0,0 +1,29 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+
+export default class SharedBreadCrumbComponent extends Component {
+  @service router;
+
+  bread = [
+    {
+      route: 'subsidy.applications',
+      crumbs: [{ label: 'Subsidiebeheer' }],
+    },
+    {
+      route: 'subsidy.detail.step.new',
+      crumbs: [{ label: 'Bekijk subsidieaanvraag' }],
+    },
+    {
+      route: 'subsidy.detail.step.detail',
+      crumbs: [{ label: 'Bekijk subsidieaanvraag' }],
+    },
```

Comment:
Silly question, but how do you get to these routes without manually editing the url?

Reply:
Try going to page 2 of the subsidies and you should be able to click through to the detail view. Not all subsidy types are available yet in the subsidiedatabank

Reply:
Wait, I thought you were going to remove the breadcrumb for the .new route? :smile:

Reply:
Oops, done here https://github.com/lblod/frontend-subsidiedatabank/pull/26
too much context-switching :sweat_smile: